src/wmtk/utils/polygon_mesh_topology_initialization.cpp

```diff
@@ -0,0 +1,477 @@
+#include "polygon_mesh_topology_initialization.h"
+#include <Eigen/Sparse>
+
+namespace wmtk {
+namespace {
+
+// Flatten a list of lists to a single vector
+VectorXl flatten(const std::vector<std::vector<long>>& list_of_lists)
+{
+    // Get size of flattened array
+    long arr_size = 0;
+    for (long i = 0; i < list_of_lists.size(); ++i) {
+        arr_size += list_of_lists[i].size();
+    }
+
+    // Build flattened array
+    VectorXl arr(arr_size);
```

Comment:
this linked code is a bit overly fancy, but if you prune the extra stuff makes this function a bit simpler:
https://github.com/mtao/core/blob/91f9bc6e852417989ed62675e2bb372e6afc7325/include/mtao/eigen/stack.hpp#L98

in particular you can use VectorXl::ConstMapType(list.data(), list.size()); to pretend each list is an Eigen Vector

Reply:
I'm not quite sure what you have in mind here. Perhaps we could discuss in person?